test/unit/models/has_content_id_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class HasContentIdTest < ActiveSupport::TestCase
```

Comment:
I found this file pretty hard to scan/parse. I think some judicious whitespace might break things up and help in that regard.